### PR TITLE
Add project for review

### PR DESCRIPTION
**Pull request by kimfom01**

### CODE REVIEW

File: ShiftLoggerApi/ShiftLoggerApi.sln [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ShiftLoggerApi", "ShiftLoggerApi\ShiftLoggerApi.csproj", "{6EB8E8DD-39A8-4477-98E1-301279073E3C}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
+		Release|Any CPU = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{6EB8E8DD-39A8-4477-98E1-301279073E3C}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{6EB8E8DD-39A8-4477-98E1-301279073E3C}.Debug|Any CPU.Build.0 = Debug|Any CPU
```
Comment (TheCSharpAcademy) [inline ShiftLoggerApi/ShiftLoggerApi.sln]: You'll need to create these two projects inside a KimFom.ShiftLogger project. So the hierarchy will be: ParentFolder > KimFom.ShiftLogger > Api, UI

**Review by TheCSharpAcademy — CHANGES_REQUESTED** [review]

**Comment by kimfom01** [comment]
Okay let me make the changes

File: ShiftLoggerApi/ShiftLoggerApi.sln (hunk 1 above)
Comment (kimfom01) [reply ShiftLoggerApi/ShiftLoggerApi.sln]: Okay let me make the changes

Comment (kimfom01) [reply ShiftLoggerApi/ShiftLoggerApi.sln]: done

File: KimFom.ShiftLogger/ShiftLoggerApi/ShiftLoggerApi/Models/Shift.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace ShiftLoggerApi.Models;
+
+public class Shift
+{
+    [Key]
+    public int Id { get; set; }
+    public string? Name { get; set; }
```
Comment (TheCSharpAcademy) [inline KimFom.ShiftLogger/ShiftLoggerApi/ShiftLoggerApi/Models/Shift.cs:9]: Looking good! Quick question about this model. Should the name ever be null?

Comment (kimfom01) [reply KimFom.ShiftLogger/ShiftLoggerApi/ShiftLoggerApi/Models/Shift.cs:9]: Lol nope. I was just trying to remove those warnings. The name shouldn't be null.

**Pull request closed without merging**